examples/src/pages/Viewer.tsx

```diff
@@ -448,11 +450,15 @@ export function Viewer() {
                   `Clicked point assigned to the object with annotationId: ${intersection.annotationId} and assetId: ${intersection?.assetRef?.id} at`,
                   point
                 );
-                if (intersection.annotationId !== 0) {
+                if (intersection.annotationId !== 0 && intersection.annotationId !== -1) {
```

Comment:
If we change `annotationId` to be 0 on DM, remember to revert this :sweat_smile: